small fix in getPlatformProxy's dispose text (#12975)
cloudflare/cloudflare-docs: content/workers/wrangler/api.md

*   `dispose()` {{<type>}}() => Promise\<void>{{</type>}}

    *   Terminates the underlying `workerd` process.

>     *   Call this after the platform proxy is no longer required by the program. If you are running a long running process (such as a dev server) that can indefinitely make use of it, you don't need to call this function.

Reviewer: ```suggestion
    *   Call this after the platform proxy is no longer required by the program. If you are running a long running process (such as a dev server) that can indefinitely make use of the proxy, you do not need to call this function.
```
